circle.yml

```diff
@@ -1,5 +1,5 @@
 test:
   override:
-    - npm run jshint
+    - node_modules/jsxhint/cli.js 'app/**/*.js' 'mock/**/*.js' 'test/**/*.js' '*.js'
```

Comment:
Why did this have to change? The `npm run jshint` command still exists... Just curious.